packages/components/src/Button/Button/_docs/Button.stickersheet.stories.tsx

```diff
@@ -1,20 +1,30 @@
 import React from "react"
 import { Meta } from "@storybook/react"
+import { ButtonProps } from "~components/Button"
```

Comment:
Oh? Weird to take it separately to `Button`. I think they should come from the same import statement.
I wonder if our pattern should be to import from `../index` or if we should import from `~components/` 🤔